### PR TITLE
docs: adding baseurl changes for regional email to the api specs

### PR DESCRIPTION
**Pull request by tbischel**
# Fixes #

SendGrid docs reference api.sendgrid.com as the URL for running the API against.  Moving forward, this will include api.eu.sendgrid.com for regionally pinned subusers.  This updates the docs for products available in the EU to point to the correct URL source.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: spec/yaml/tsg_domain_authentication_v3.yaml [hunk 1]
```diff
@@ -1636,18 +1638,18 @@ components:
...
-              - legacy
-              - user_id
-              - custom_spf
-              - id
               - automatic_security
-              - valid
-              - username
-              - dns
-              - subdomain
+              - custom_spf
```
Comment (tbischel) [inline spec/yaml/tsg_domain_authentication_v3.yaml:1642]: my changes didn't include any updates to the parameters

File: spec/yaml/tsg_email_activity_v3.yaml [hunk 2]
```diff
@@ -337,16 +337,16 @@ paths:
                     description: Whether or not the outbound IP is dedicated vs shared
                     $ref: '#/components/schemas/OutboundIpType'
                 required:
+                - api_key_id
```
Comment (tbischel) [inline spec/yaml/tsg_email_activity_v3.yaml:340]: parameter changes not mine

File: spec/yaml/tsg_lmc_contactdb_v3.yaml [hunk 3]
```diff
@@ -411,19 +411,14 @@ paths:
       in: path
       required: true
       schema:
-        type: string
```
Comment (tbischel) [inline spec/yaml/tsg_lmc_contactdb_v3.yaml:414]: parameter changes not mine

File: spec/yaml/tsg_mc_contacts_v3.yaml [hunk 4]
```diff
@@ -358,8 +358,8 @@ paths:
               properties:
                 identifier_type:
                   type: string
-                  description: 'A type of the following: EMAIL, PHONENUMBERID, EXTERNALID,
```
Comment (tbischel) [inline spec/yaml/tsg_mc_contacts_v3.yaml:361]: changes not mine

File: spec/yaml/tsg_mc_custom_fields_v3.yaml [hunk 5]
```diff
@@ -100,9 +100,9 @@ paths:
                   _metadata:
                     $ref: '#/components/schemas/Metadata'
                 required:
-                - name
```
Comment (tbischel) [inline spec/yaml/tsg_mc_custom_fields_v3.yaml:103]: not mine

File: spec/yaml/tsg_mc_senders_v3.yaml [hunk 6]
```diff
@@ -97,11 +97,11 @@ paths:
                   type: string
                   description: The country of the Sender.
               required:
-              - country
```
Comment (tbischel) [inline spec/yaml/tsg_mc_senders_v3.yaml:100]: not mine

File: spec/yaml/tsg_scheduled_sends_v3.yaml [hunk 7]
```diff
@@ -288,8 +290,8 @@ components:
           description: The status of the scheduled send.
           $ref: '#/components/schemas/Status2'
       required:
-      - status
```
Comment (tbischel) [inline spec/yaml/tsg_scheduled_sends_v3.yaml:291]: not mine

File: spec/yaml/tsg_sso_v3.yaml [hunk 8]
```diff
@@ -1617,8 +1619,8 @@ components:
             - id
             - permission_type
       required:
-      - first_name
       - email
+      - first_name
```
Comment (tbischel) [inline spec/yaml/tsg_sso_v3.yaml:1623]: not mine

File: spec/yaml/tsg_templates_v3.yaml [hunk 9]
```diff
@@ -489,10 +491,10 @@ components:
         warning:
           $ref: '#/components/schemas/TransactionalTemplateWarning'
       required:
-      - name
```
Comment (tbischel) [inline spec/yaml/tsg_templates_v3.yaml:492]: not mine

**Review by CristhianMotoche — COMMENTED** [review]
Hey @tbischel Why only the `.yaml`. file changed? Shouldn't the `.json` files be updated, too?

**Review by bermudezmt — APPROVED** [review]

File: spec/yaml/tsg_email_activity_v3.yaml (hunk 2 above)
Comment (bermudezmt) [reply spec/yaml/tsg_email_activity_v3.yaml:340]: @CristhianMotoche do you know why these changes are getting transpiled?

Comment (CristhianMotoche) [reply spec/yaml/tsg_email_activity_v3.yaml:340]: Hey @tbischel @bermudezmt I'm not really sure but I'm guessing something was added in the transpiler that updates this list in alphabetical order.

**Pull request merged**